benchmark/parse.yaml

```diff
@@ -0,0 +1,57 @@
+loop_count: 100
+contexts:
+  - gems:
+      rexml: 3.2.6
+    require: false
+    prelude: require 'rexml'
+  - name: 3.2.6(YJIT)
+    gems:
+      rexml: 3.2.6
+    require: false
+    prelude: |
+      require 'rexml'
+      RubyVM::YJIT.enable
+  - name: master
+    prelude: |
+      $LOAD_PATH.unshift(File.expand_path("lib"))
+      require 'rexml'
+  - name: master(YJIT)
+    prelude: |
+      $LOAD_PATH.unshift(File.expand_path("lib"))
+      require 'rexml'
+      RubyVM::YJIT.enable
+
+prelude: |
+  require 'rexml/document'
+  require 'rexml/parsers/sax2parser'
+  require 'rexml/parsers/pullparser'
+  require 'rexml/parsers/streamparser'
+  require 'rexml/streamlistener'
+
+  n_elements = Integer(ENV.fetch("N_ELEMENTS", "10000"), 10)
+  n_attributes = Integer(ENV.fetch("N_ATTRIBUTES", "10"), 10)
```

Comment:
With these configurations, we take 10+ minutes for one `rake benchmark`: https://github.com/ruby/rexml/actions/runs/7431570516/job/20222471649?pr=104

Can we reduce benchmark time? Or do we need these numbers for meaningful benchmark?

Reply:
I set this value because I need to execute XML with a large number of elements, but the execution time is too long, so I will reduce the number of elements.

Reply:
https://github.com/ruby/rexml/actions/runs/7434389228/job/20228488163?pr=104
Reduced to 1/4.